src/Microsoft.AspNet.Diagnostics.Elm/ElmScope.cs

```diff
@@ -2,9 +2,10 @@
 #if DNX451
 using System.Runtime.Remoting;
 using System.Runtime.Remoting.Messaging;
-#endif
+#else
 using System.Threading;
```

Comment:
This was used only in the #else case. So VS intellisense is always showing this is an unused using statement. So moved it into the #else to clean up the intellisense warning.